MonoGame.Framework/Input/Mouse.cs

```diff
@@ -24,11 +24,10 @@ public static IntPtr WindowHandle
             {
                 return PlatformGetHandle();
             }
-#pragma warning disable RECS0029 // Warns about property or indexer setters and event adders or removers that do not use the value parameter
             set
-#pragma warning restore RECS0029 // Warns about property or indexer setters and event adders or removers that do not use the value parameter
             {
                 // only for XNA compatibility, yet
+                IntPtr temp = value;
```

Comment:
`RECS0029` was not understood by the Visual Studio compiler.  It can't even be `CS0029` because that is a different compiler error, not warning.  Is this a MonoDevelop-only compiler warning?